tests/conftest.py

```
@@ -726,6 +726,15 @@ async def get_smart(self) -> Any:
                 """Return an empty SMART payload for coordinator tests."""
                 return []
 
+            async def get_smart_info(self, device: str, info_type: str = "a") -> Any:
```

Ruff (0.15.17)

[warning] Unused method argument: `device`

(ARG002)